app/lib/uploadToThirdParty.ts

```diff
@@ -0,0 +1,41 @@
+import { Project } from '@stackblitz/sdk'
+import * as LZString from 'lz-string'
+
+export function createStackBlitzProject(html: string) {
+	const stacklitzProject: Project = {
+		title: 'Make real from tldraw',
+		description: 'Your AI generated example made at https://makereal.tldraw.com/',
+		template: 'html',
+		files: { 'index.html': html },
+	}
+	return stacklitzProject
+}
+
+export function getCodeSandboxUrl(html: string) {
+	const project = createCodeSandboxProject(html)
+	return `https://codesandbox.io/api/v1/sandboxes/define?parameters=${project}`
+}
+
+// The following two functions are from
```

Comment:
Didn't want to depend on the whole package so I just copied these two functions. From my understanding we should be fine since we are using AGPL 3.0, which is compatible with GPL 3.